SkyEditor.RomEditor.Rtdx/Domain/Rtdx/Structures/MessageBinEntry.cs

```diff
@@ -24,8 +24,11 @@ public MessageBinEntry(IReadOnlyBinaryDataAccessor data)
                 var stringOffset = sir0.Data.ReadInt64(entryOffset);
                 var hash = sir0.Data.ReadInt32(entryOffset + 8);
                 var unknown = sir0.Data.ReadInt32(entryOffset + 0xC);
-                strings.Add(hash, sir0.Data.ReadNullTerminatedUnicodeString(stringOffset));
-                hashes.Add(stringOffset, hash);
+                if (!strings.ContainsKey(hash)) // script.bin contains duplicate hashes
```

Comment:
Wat? Do we need to preserve these duplicates? Can we even use a dictionary?

Reply:
No idea, I only included the check to avoid crashes. I haven't checked the content of those strings.